cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -1451,6 +1451,8 @@ struct DispatchRadixSort :
                     }
                 }
 
+                if (CubDebug(error)) break;
```

Comment:
Is this check needed?

Reply:
The check is necessary to break out of the loop in case of error. An extra `CubDebug()` is not, so I removed it.